code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```diff
@@ -947,8 +948,18 @@ func (fsh *StorageHandler) WriteFile(ctx context.Context, r *http.Request) (*blo
 	}
 
 	allocationID := allocationObj.ID
-	fileOperation := getFileOperation(r)
-	existingFileRef := getExistingFileRef(fsh, ctx, r, allocationObj, fileOperation)
+	cmd := createFileCommand(r)
+	err2 := cmd.IsAuthorized(ctx, r, allocationObj, clientID)
```

Comment:
the reason I'm checking the err2 later is due to multiple system, unit and integration tests failures. since we are calling the IsAuthorized much ahead now to reduce db requests.

Reply:
If we got error from `IsAuthorized()` and do not check and return early, the execution of the following `IsACollaborator()`, `verifySignatureFromRequest()` would be a waste of time/resources, and moving `IsAuthorized()` ahead would not make sense. 

We should update unit tests and system tests instead of posting the the error check.

Reply:
Please fix this